### PR TITLE
HaskellNet, HaskellNet-SSL: jailbreak to fix dependencies versions

### PR DESCRIPTION
**Pull request by ghost**
HaskellNet, HaskellNet-SSL: jailbreak to fix dependencies versions

### CODE REVIEW

**Comment by cdepillabout** [comment]
@tjilerom Thanks for sending this PR.

Could you rebase this on top of `haskell-updates` so the history is easy to understand?

File: pkgs/development/haskell-modules/configuration-common.nix [hunk 1]
```diff
@@ -142,26 +142,26 @@ self: super: {
   vector = dontCheck (if pkgs.stdenv.isi686 then appendConfigureFlag super.vector "--ghc-options=-msse2" else super.vector);
 
   conduit-extra = if pkgs.stdenv.isDarwin
-    then super.conduit-extra.overrideAttrs (drv: { __darwinAllowLocalNetworking = true; })
-    else super.conduit-extra;
+                  then super.conduit-extra.overrideAttrs (drv: { __darwinAllowLocalNetworking = true; })
```
Comment (peti) [inline pkgs/development/haskell-modules/configuration-common.nix]: Could you please explain a bit what this addition is about? Why is that attribute required?

File: pkgs/development/haskell-modules/configuration-common.nix [hunk 2]
```diff
@@ -142,26 +142,26 @@ self: super: {
...
   # Hakyll's tests are broken on Darwin (3 failures); and they require util-linux
   hakyll = if pkgs.stdenv.isDarwin
-    then dontCheck (overrideCabal super.hakyll (drv: {
-      testToolDepends = [];
-    }))
-    else super.hakyll;
+           then dontCheck (overrideCabal super.hakyll (drv: {
+             testToolDepends = [];
+           }))
+           else super.hakyll;
```
Comment (peti) [inline pkgs/development/haskell-modules/configuration-common.nix]: What is this change about? This looks like you're changing only whitespace.

File: pkgs/development/haskell-modules/configuration-common.nix [hunk 3]
```diff
@@ -142,26 +142,26 @@ self: super: {
...
 
   double-conversion = if !pkgs.stdenv.isDarwin
-    then super.double-conversion
-    else addExtraLibrary super.double-conversion pkgs.libcxx;
+                      then super.double-conversion
+                      else addExtraLibrary super.double-conversion pkgs.libcxx;
```
Comment (peti) [inline pkgs/development/haskell-modules/configuration-common.nix]: ditto

**Review by peti — CHANGES_REQUESTED** [review]
Please commit unrelated whitespace changes.

**Comment by peti** [comment]
Closing because of inactivity.

**Pull request closed without merging**